README.md

```diff
@@ -87,6 +87,10 @@ Available options/variables and their default values:
 | GOG_EMAIL     	|         	| GOG email for login. Overrides EMAIL.                                  	|
 | GOG_PASSWORD  	|         	| GOG password for login. Overrides PASSWORD.                            	|
 | GOG_NEWSLETTER	| 0       	| Do not unsubscribe from newsletter after claiming a game if 1.         	|
+| GOG_GIVEAWAY	  | 1       	| Claims giveaway game(s). Is enabled by default. |
+| GOG_FREEGAMES	  | 0      	| Claims other free games that are not demos or prologue games. Is disabled by default.         	|
+| GOG_FREEGAMES_URL  | [freegames_url](https://www.gog.com/en/games?priceRange=0,0&languages=en&order=asc:title&hideDLCs=true&excludeTags=demo&excludeTags=freegame) | URL to get games to claim additionally. You can add filters for language or certain categories that you don't like. The filter to hide owned games (hideOwned=true) is automatically added so please do not add it.|
```

Comment:
Does something bad happen if you add another `hideOwned=true`?
If yes, could filter it out. Then remove 'The filter to hide owned games (hideOwned=true) is automatically added so please do not add it.'.